core/esmf-aspect-meta-model-java/src/main/java/org/eclipse/esmf/metamodel/Namespace.java

```diff
@@ -0,0 +1,9 @@
+package org.eclipse.esmf.metamodel;
+
+/**
+ * A Namespace is Structure Element for describing namespace and providing user readable information about Namespace.
+ *
+ * @since SAMM 2.2.0
+ */
+public interface Namespace extends StructureElement {
```

Comment:
This should not extends StructureElement; since it does not have Properties like Aspects and Entities.